source/matplot/core/axes_type.cpp

```diff
@@ -2745,9 +2743,9 @@ namespace matplot {
     }
 
     std::vector<function_line_handle>
-    axes_type::fplot(std::vector<function_line::function_type> equations,
-                     std::array<double, 2> x_range,
-                     std::vector<std::string> line_specs) {
+    axes_type::fplot(const std::vector<function_line::function_type> &equations,
```

Comment:
Same here. All these API changes are OK.

Reply:
It's OK if the thing is recompiled, but I am not sure at ABI level, and I do not know if anyone cares.